nupic/research/frameworks/pytorch/datasets/gsc_factory.py

```diff
@@ -57,9 +59,18 @@ def preprocessed_gsc(root, train=True, download=True):
     if train:
         basename = "gsc_train"
         qualifiers = range(30)
```

Comment:
Can you make the range a parameter, defaulting to 30? We have 100 in our internal version.